src/scripts/components/Play.js

```diff
@@ -540,86 +565,93 @@ class Play extends Component<Props, State> {
     } else {
       return (
         <Wrapper isEmbed={isEmbed}>
-          <PlayerWrapper
```

Comment:
This big diff is kind of scary, but if the new player works it works!

Is this mostly because the player is now wrapped in `VideoWrapper`?

Reply:
@bent0b0x yes. Basically it's just about the wrapper from `/play` and `/embed`